README.md

```diff
@@ -1 +1,33 @@
-# kitcat
\ No newline at end of file
+# KitCat
+
+*Sometimes schema migrations are just not enough.*
+
+Data migration framework written in plain Ruby. It is pretty generic, since it does not depend on any web framework.
+
+## Features of Framework
+
+This is a small migration framework that offers the following functionality for free:
+
+1. Logging
+2. Progress Bar
+3. Gracefully handling when user interrupts (Ctrl+C or `kill <process_id>`)
+
+## Example Usage
+
+Assuming that the migration strategy is implemented with a class `MigrationStrategy`, then executing a migration is quite simple:
+
+``` ruby
+migration_strategy  = MigrationStrategy.new
+migration_framework = Migrations::Framework.new(migration_strategy)
```

Comment:
We've changed the namespace to KitCat. Should we merge this and change later or do now?